Managers/UInputMan.cpp

```diff
@@ -357,11 +356,86 @@ void UInputMan::InputScheme::SetPreset(int schemePreset)
+        m_aInputMapping[INPUT_L_LEFT].SetKey(KEY_LEFT);
+        m_aInputMapping[INPUT_L_RIGHT].SetKey(KEY_RIGHT);
+        m_aInputMapping[INPUT_FIRE].SetKey(KEY_1_PAD);
+        m_aInputMapping[INPUT_AIM].SetKey(KEY_2_PAD);
+        m_aInputMapping[INPUT_AIM_UP].SetKey(KEY_UP);
+        m_aInputMapping[INPUT_AIM_DOWN].SetKey(KEY_DOWN);
+        m_aInputMapping[INPUT_PIEMENU].SetKey(KEY_3_PAD);
+        m_aInputMapping[INPUT_JUMP].SetKey(KEY_ENTER_PAD);
+        m_aInputMapping[INPUT_CROUCH].SetKey(KEY_DEL_PAD);
+        m_aInputMapping[INPUT_NEXT].SetKey(KEY_5_PAD);
+        m_aInputMapping[INPUT_PREV].SetKey(KEY_4_PAD);
+        m_aInputMapping[INPUT_WEAPON_RELOAD].SetKey(KEY_0_PAD);
+        m_aInputMapping[INPUT_WEAPON_DROP].SetKey(KEY_6_PAD);
+        m_aInputMapping[INPUT_WEAPON_PICKUP].SetKey(KEY_9_PAD);
+        m_aInputMapping[INPUT_WEAPON_CHANGE_PREV].SetKey(KEY_7_PAD);
+        m_aInputMapping[INPUT_WEAPON_CHANGE_NEXT].SetKey(KEY_8_PAD);
+
+    case PRESET_P3DEFAULT:
+        m_aInputMapping[INPUT_L_UP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_L_DOWN].SetKey(KEY_S);
+        m_aInputMapping[INPUT_L_LEFT].SetKey(KEY_A);
+        m_aInputMapping[INPUT_L_RIGHT].SetKey(KEY_D);
+        m_aInputMapping[INPUT_R_UP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_R_DOWN].SetKey(KEY_S);
+        m_aInputMapping[INPUT_R_LEFT].SetKey(KEY_A);
+        m_aInputMapping[INPUT_R_RIGHT].SetKey(KEY_D);
+        m_aInputMapping[INPUT_FIRE].SetKey(KEY_H);
+        m_aInputMapping[INPUT_AIM].SetKey(KEY_J);
+        m_aInputMapping[INPUT_AIM_UP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_AIM_DOWN].SetKey(KEY_S);
+        m_aInputMapping[INPUT_PIEMENU].SetKey(KEY_K);
+        m_aInputMapping[INPUT_JUMP].SetKey(KEY_L);
+        m_aInputMapping[INPUT_CROUCH].SetKey(KEY_STOP);
+        m_aInputMapping[INPUT_NEXT].SetKey(KEY_U);
+        m_aInputMapping[INPUT_PREV].SetKey(KEY_Y);
+
+    case PRESET_P4DEFAULT:
+        m_aInputMapping[INPUT_L_UP].SetKey(KEY_UP);
+        m_aInputMapping[INPUT_L_DOWN].SetKey(KEY_DOWN);
+        m_aInputMapping[INPUT_L_LEFT].SetKey(KEY_LEFT);
+        m_aInputMapping[INPUT_L_RIGHT].SetKey(KEY_RIGHT);
+        m_aInputMapping[INPUT_R_UP].SetKey(KEY_UP);
+        m_aInputMapping[INPUT_R_DOWN].SetKey(KEY_DOWN);
+        m_aInputMapping[INPUT_R_LEFT].SetKey(KEY_LEFT);
+        m_aInputMapping[INPUT_R_RIGHT].SetKey(KEY_RIGHT);
+        m_aInputMapping[INPUT_FIRE].SetKey(KEY_1_PAD);
+        m_aInputMapping[INPUT_AIM].SetKey(KEY_2_PAD);
+        m_aInputMapping[INPUT_AIM_UP].SetKey(KEY_UP);
+        m_aInputMapping[INPUT_AIM_DOWN].SetKey(KEY_DOWN);
+        m_aInputMapping[INPUT_PIEMENU].SetKey(KEY_3_PAD);
+        m_aInputMapping[INPUT_JUMP].SetKey(KEY_DEL_PAD);
+        m_aInputMapping[INPUT_CROUCH].SetKey(KEY_STOP);
+        m_aInputMapping[INPUT_NEXT].SetKey(KEY_5_PAD);
+        m_aInputMapping[INPUT_PREV].SetKey(KEY_4_PAD);
 
-    }
     // Some generic defaults; no real preset is set
-    else
-    {
+    default:
```

Comment:
`PRESET_NONE` is not a `default` here.
`default` would be 
```
RTEAbort("Undefined input preset value passed in. See InputPreset enumeration for defined values.");
```

Reply:
Actually no, i'm derping. Ignore this.

Reply:
didn't see this so i already committed on your previous comment.
do you mean it should be as it was when you commented?

Reply:
Yeah it was correct before, sorry about that, you can change it back.
Also the indentation in the `PRESET_XBOX360` is screwy because you're mixing spaces and tabs.

The codebase is using spaces for indentation but that is heresy, so we're correcting it to tabs whenever we work on something.
Generally it isn't an issue if the whole file hasn't been reformatted yet but it's better to do that anyway.
Make sure you set VS to keep tabs instead of replacing them with spaces.

Reply:
ok, reverted it.
also i think i got the spaces into tabs now.